sfge/src/test/java/com/salesforce/rules/usewithsharingondatabaseoperation/BaseUseWithSharingOnDatabaseOperationTest.java

```diff
@@ -66,15 +66,16 @@ public abstract class BaseUseWithSharingOnDatabaseOperationTest extends BasePath
         "Database.getAsyncLocator(Database.deleteAsync(List.of(a), false))",
         "Database.getAsyncSaveResult(Database.insert(a))",
         "Database.getDeleted('account__c', Datetime.now().addHours(-1), Datetime.now())",
-        "Database.getQueryLocator('SELECT Id, NumberOfEmployees FROM account WHERE NumberOfEmployees = 3 LIMIT 1')",
-        "Database.getQueryLocatorWithBinds('SELECT Id, NumberOfEmployees FROM account WHERE NumberOfEmployees = 3 LIMIT 1', new Map<String, Object>{'name' => 'KENSINGTON'})",
+        "Database.getQueryLocator('SELECT Name, NumberOfEmployees FROM account WHERE NumberOfEmployees = 3 LIMIT 1')",
+        "Database.getQueryLocatorWithBinds('SELECT Name, NumberOfEmployees FROM account WHERE NumberOfEmployees = :i LIMIT 1', new Map<String, Object>{'i' => 3})",
```

Comment:
Updated these because Josh mentioned `SELECT Id, .. ` is not good to have, possibly an error. Also, I made the original binds when I didn't really know what they were, and they didn't make sense.